awscli/customizations/scalarparse.py

```diff
@@ -36,13 +38,27 @@ def identity(x):
     return x
 
 
+def choose_default_formatters(timestamp_format):
+    parsers = dict(blob_parser=identity)
+    if timestamp_format == 'none':
+        # For backwards compatibility reasons, we replace botocore's timestamp
+        # parser (which parsers to a datetime.datetime object) with the
```

Comment:
Small nit, ``parses`` not ``parsers`` in "which parsers to a datetime.datetime object".